### DISCUSSION TITLE
two questions.

### DISCUSSION THREAD

**Discussion by oda9** [alvarobartt/investpy#401] [Q&A] [answered]
I have two questions.

First.
Is it possible to use get_crypto_recent_data to get the value of JPY bitcoin?

Second.
There are some stocks whose dividend data cannot be retrieved by get_stock_dividends.
Is there a way to get it?

Example code I tried
`investpy.get_stock_dividends(stock='RPRX',country='united states')`

**Comment by alvarobartt** [comment, marked as answer]
Hi @oda9, first of all sorry for answering too late! :weary:

I've moved this issue to a Q&A discussion.

Regarding your first question, currerntly the static data for cryptos is pretty limited and just the USD conversion is offered, but you can use `investpy.search_quotes()` to search the crypto in the currency that you want, and then retrieve the data.

So `investpy.get_crypto_recent_data()` won't work since it just contains the main crypto currencies in USD, but using the following piece of code will work:

```python
import investpy

search_result = investpy.search_quotes(text='BTC JPY', products=['cryptos'], n_results=1)

recent_data = search_result.retrieve_recent_data()
historical_data = search_result.retrieve_historical_data(from_date='01/01/2020', to_date='01/01/2021')
```

But take into consideration that Investing.com offers different exchanges, which means that the first result may not be the information that you are looking for, so lets assume that you want the information provided by the exchange "BTCBOX" not the one provided by "bitFlyer" which is the first entry, then you could for example use the following piece of code:

```python
import investpy

search_results = investpy.search_quotes(text='BTC JPY', products=['cryptos'])

for search_result in search_results:
    if search_result.exchange == 'BTCBOX':
        print('BTC/JPY from BTCBOX available at Investing.com!')
        break

recent_data = search_result.retrieve_recent_data()
historical_data = search_result.retrieve_historical_data(from_date='01/01/2020', to_date='01/01/2021')
```

__Note__ that `investpy.search_quotes` uses Investing.com search engine, which means that the results provided are the same ones than the ones shown in the website. Also make sure that you have the latest version, since `investpy` has been recently upgraded to `v1.0.7` :fire: adding a lot of fixes and new features to `investpy.search_quotes` as described in the release notes at https://github.com/alvarobartt/investpy/releases/tag/1.0.7

Also for more information regarding the search functionality, you should check the documentation at https://investpy.readthedocs.io/_info/usage.html#search-live-data

And, regarding your second question, could you please open an issue describing the error that you are running into so that I can fix it before the next release `v1.0.8` comes out?

Thanks a lot and hope this was useful to you! :hugs:

**Reply by oda9** [reply]
Hi @alvarobartt, Thanks for the reply.

As for crypto, we were able to acquire it.
I would also like to raise an issue regarding your second question.

Thanks for the very useful API.
